apps/editor/src/app/routes/dashboard.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled'
+import {useMeQuery} from '@wepublish/editor/api'
+import {useMemo} from 'react'
+import {useTranslation} from 'react-i18next'
+import {FlexboxGrid, Panel as RPanel} from 'rsuite'
+
+import {ActivityFeed} from '../atoms/dashboard/activityFeed'
+import {SubscriberChart} from '../atoms/dashboard/subscriberChart'
+import {PermissionControl} from '../atoms/permissionControl'
+import {ListViewContainer, ListViewHeader} from '../ui/listView'
+
+const Wrapper = styled(FlexboxGrid)`
+  margin-top: 20px;
+  height: calc(100vh - 220px);
+`
+
+const Item = styled(FlexboxGrid.Item)`
+  max-height: 100%;
+`
+
+export function Dashboard() {
+  const {t} = useTranslation()
+
+  const {data: me} = useMeQuery()
+  const name = useMemo(
+    () => me?.me?.preferredName ?? me?.me?.firstName ?? me?.me?.name ?? t('dashboard.user'),
+    [me]
```

Comment:
t is also needed in here technically.

But we can go without useMemo here as the CPU impact isn't big enough to save it in the memory.

Reply:
Not sure what you mean with t?